frontend/src/pages/miscellaneous/ProfilePage.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import axios from 'axios';
```

Comment:
Handling of axios should not be here, should be handled under src/services/user-service